### PR TITLE
setup.cfg: flake8 is a superset of pycodestyle

### PR DESCRIPTION
**Pull request by cclauss**
~Blocked by #500~

~Use __print()__ function in both Python 2 and Python 3.~
~https://travis-ci.org/fredrik-johansson/mpmath/jobs/617056073#L246~

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -21,6 +21,7 @@ script:
   # Guard against invalid escapes in strings, like '\s'.
   - python -We:invalid -m compileall -f mpmath -q
   - pycodestyle
+  - flake8 . --count --select=E9,F63,F7 --show-source --statistics
```
Comment (skirpichev) [inline .travis.yml:24]: 1) You might put most flake8 args to setup.cfg
2) run pycodestyle on the line above will be redundant, I think?

File: setup.cfg [hunk 2]
```diff
@@ -13,10 +13,10 @@ classifiers = License :: OSI Approved :: BSD License
               Programming Language :: Python :: 2
               Programming Language :: Python :: 2.7
               Programming Language :: Python :: 3
-              Programming Language :: Python :: 3.4
```
Comment (skirpichev) [inline setup.cfg:16]: Why is this?  Travis-CI on your branch - still has tests for CPython 3.4.

**Review by skirpichev — CHANGES_REQUESTED** [review]

**Comment by skirpichev** [comment]
#500 was merged.  Perhaps, you can merge the master branch to this branch to be sure - tests pass.

File: setup.cfg (hunk 2 above)
Comment (cclauss) [reply setup.cfg:16]: Python 3.4 and 3.5 are end-of-life.
https://devguide.python.org/devcycle/#end-of-life-branches
https://devguide.python.org/#status-of-python-branches

Comment (skirpichev) [reply setup.cfg:16]: 2.7 - too, and?

**Pull request closed without merging**